### PR TITLE
Support show trial command on (remote | reuse) mode

### PR DESCRIPTION
**Pull request by Lijiaoa**
remote, reuse, aml:
image

local:
image

### CODE REVIEW

File: src/webui/src/components/overview/command/Command2.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const Command2 = (): any => {
 
     if (clusterMetaData !== undefined) {
         for (const item of clusterMetaData) {
-            if (item.key === 'command') {
+            if (item.key === 'command' || item.key === 'trial_config') {
                 trialCommand = item.value;
```
Comment (SparkSnail) [inline src/webui/src/components/overview/command/Command2.tsx:14]: the data format in trial_config is {"command":"python3..", "gpuNum":"",...}, should use item.value.command for trial_config.

Comment (Lijiaoa) [reply src/webui/src/components/overview/command/Command2.tsx:14]: will fix

File: src/webui/src/components/overview/command/Command2.tsx [hunk 2]
```diff
@@ -11,7 +11,10 @@ export const Command2 = (): any => {
     if (clusterMetaData !== undefined) {
         for (const item of clusterMetaData) {
             if (item.key === 'command') {
-                trialCommand = item.value;
+                trialCommand = item.value as string;
+            }
+            if (item.key === 'trial_config') {
+                trialCommand = item.value['command'];
```
Comment (SparkSnail) [inline src/webui/src/components/overview/command/Command2.tsx:17]: if 'command' field does not exist in trial_config, will this line cause webui crash?

Comment (ultmaster) [reply src/webui/src/components/overview/command/Command2.tsx:17]: Why not `item.value.command`?

Comment (Lijiaoa) [reply src/webui/src/components/overview/command/Command2.tsx]: had fix this question

**Review by ultmaster — APPROVED** [review]

File: src/webui/src/static/interface.ts [hunk 3]
```diff
@@ -184,10 +184,17 @@ interface ExperimentParams {
...
-        value: string;
+        value: string | ClusterItem;
     }[];
 }
 
+interface ClusterItem {
+    command?: string;
+    codeDir?: string;
+    image?: string;
+    gpuNum?: number;
```
Comment (SparkSnail) [inline src/webui/src/static/interface.ts:195]: There might be more items trial_config, like 'virtualCluster', 'memoryMB', 'paiStorageConfigName', will this work if more items added in trial_config?

**Review by SparkSnail — APPROVED** [review]

File: src/webui/src/static/interface.ts (hunk 3 above)
Comment (Lijiaoa) [reply src/webui/src/static/interface.ts]: test, no problem

**Review by 961974 — APPROVED** [review]

**Pull request merged**